src-docs/src/views/inline_edit/inline_edit_example.js

```diff
@@ -80,6 +84,26 @@ export const InlineEditExample = {
       demo: <InlineEditTitle />,
       props: { EuiInlineEditTitle },
     },
+    {
+      title: 'Retrieving edited text',
```

Comment:
Minor copy suggestion - I think it makes more sense to have the section title match the prop it describes
```suggestion
      title: 'Saving edited text',
```